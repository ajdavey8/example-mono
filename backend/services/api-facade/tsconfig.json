{
  "extends": "@tsconfig/node18/tsconfig.json",
  "compilerOptions": {
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "forceConsistentCasingInFileNames": true,
    "isolatedModules": true,
    "noEmit": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "skipLibCheck": true,
    "sourceMap": true,
    "strictNullChecks": true
  },
  "include": ["stack/**/*.ts", "src/**/*.ts"]
}