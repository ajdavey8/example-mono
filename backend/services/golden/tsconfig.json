{
  "extends": "../../foundation/tsconfig.json",
  "include": ["src/**/*.ts"],
  "references": [
    {
      "path": "../../lib/events"
    },
    {
      "path": "../../lib/logging"
    },
    {
      "path": "../../lib/types"
    },
    {
      "path": "../../lib/util"
    },
    {
      "path": "../../lib/error"
    },
    {
      "path": "../../lib/http"
    },
    {
      "path": "../../lib/middleware"
    },
    {
      "path": "../../foundation/testing"
    }
  ]
}
